src/components/StatusBadge/index.tsx

```diff
@@ -36,6 +36,12 @@ const StatusBadge: React.FC<StatusBadgeProps> = ({ status }) => {
           {intl.formatMessage(globalMessages.pending)}
         </Badge>
       );
+    case MediaStatus.UNKNOWN:
+      return (
+        <Badge badgeType="danger">
+          {intl.formatMessage(globalMessages.failed)}
+        </Badge>
+      );
```

Comment:
This will cause the failed status badge to show up in many place throughout the app, not just the RequestList.

RequestCard uses this. So does the movie/tv details pages now.